### PR TITLE
chore(CICD): GitHub Action to generate embeddings

### PR DESCRIPTION
**Pull request by ThisIsDemetrio**
## Description

The PR includes a GitHub Action to call a specific GitLab pipeline dedicated to the generation of embeddings to be stored in a MongoDB Database for execution of [vector searches](https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-overview/).

The Action is meant to be triggered manually, by those who have a sufficient role.

## Pull Request Type

- [ ] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [X] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [X] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [X] No sensitive content has been committed

### CODE REVIEW

File: .github/workflows/generate-embeddings.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: eic/trigger-gitlab-ci@v3
+        with:
+          url: https://git.tools.mia-platform.eu
+          project_id: ${{ vars.MIA_ASSISTANT_GITLAB_PROJECT_ID }}
+          token: ${{ secrets.MIA_ASSISTANT_GITLAB_TOKEN }}
+          variables: |
+            GENERATE_EMBEDDINGS=true
+          ref_name: master
```
Comment (fredmaggiowski) [inline .github/workflows/generate-embeddings.yml:17]:
```suggestion
          ref_name: master

```

File: .github/workflows/generate-embeddings.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+jobs:
+  generate-embeddings:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: eic/trigger-gitlab-ci@v3
+        with:
+          url: https://git.tools.mia-platform.eu
+          project_id: ${{ vars.MIA_ASSISTANT_GITLAB_PROJECT_ID }}
+          token: ${{ secrets.MIA_ASSISTANT_GITLAB_TOKEN }}
```
Comment (fredmaggiowski) [inline .github/workflows/generate-embeddings.yml:14]: This token should be created before merging the PR @davidebianchi  FYI

Comment (fredmaggiowski) [reply .github/workflows/generate-embeddings.yml:14]: actually we can merge and only wait for the trigger but let's make sure the secret name is ok

Comment (davidebianchi) [reply .github/workflows/generate-embeddings.yml:14]: Added to the repo

**Review by davidebianchi — APPROVED** [review]

File: .github/workflows/generate-embeddings.yml (hunk 2 above)
Comment (ThisIsDemetrio) [reply .github/workflows/generate-embeddings.yml:14]: Tokens have been created.

**Pull request merged**